src/Raven.Server/Documents/Indexes/IndexField.cs

```diff
@@ -118,6 +129,9 @@ public override int GetHashCode()
                 hashCode = (hashCode * 397) ^ (int)Indexing;
                 hashCode = (hashCode * 397) ^ (int)TermVector;
                 hashCode = (hashCode * 397) ^ (HasSuggestions ? 233 : 343);
+                hashCode = (hashCode * 397) ^ (Spatial?.GetHashCode() ?? 0);
+                hashCode = (hashCode * 397) ^ (Vector?.GetHashCode() ?? 0);
+                //todo: vector & spatial as well?
```

Comment:
IMO yes

Reply:
@maciejaszyk ? You marked it as resolved but TODO is still here

Reply:
TODO ^^